docs/data-integrations/duckdb.mdx

```diff
@@ -39,6 +39,6 @@ WITH
 You can use this established connection to query your table as follows:
 
 ```sql
-SELECT *
+SELECT *;
```

Comment:
The semicolon should be at the end of the statement and not in its first line. Please correct it.